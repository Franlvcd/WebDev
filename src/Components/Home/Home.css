.home-background {
  min-height: 100vh;
  background-image: url('https://cdn.dribbble.com/users/88525/screenshots/2534623/earth.gif');
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
}

.title-background {
  background-color: rgba(125, 197, 125, 0.9);
  display: flex;
  justify-content: center;
  /* width: 50%; */
  margin-top: 5px;
  margin-bottom: 15px;
}

.font-link {
  font-family: 'Roboto Slab', cursive;
}

.title-colour-layer {
  width: 100%;
  height: 100%;
  background-color: rgba(0, 0, 0, 0.4);
  padding-top: 22%;
}

.title-text {
  color: white;
  text-align: center;
  padding-bottom: 48%;
  font-size: 50px;
  font-weight: bold;
}

.blank-body {
  padding: 24px;
  min-height: 500px;
  background-color: white;
}
