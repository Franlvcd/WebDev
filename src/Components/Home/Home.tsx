import './Home.css'

export const Home = () => {
  return (
    <div>
      <div className="home-background">
        <div className="title-colour-layer">
          <div style={{ display: 'flex', justifyContent: 'center' }}>
            <div className="title-text">
              <div className="title-background">
                Care to make your solution more eco-friendly?
              </div>
              <div>Greener can help!</div>
            </div>
          </div>
        </div>
      </div>
      {/* <div className="blank-body"></div> */}
    </div>
  )
}
