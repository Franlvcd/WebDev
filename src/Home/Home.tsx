import './Home.css'
import { Typography } from 'antd'

const { Title } = Typography

export const Home = () => {
  return (
    <div className="home">
      <Title>Next Source of Clean Water</Title>
    </div>
  )
}
