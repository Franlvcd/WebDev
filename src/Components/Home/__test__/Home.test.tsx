import ReactDOM from 'react-dom'
import { Home } from '../Home'
import '@testing-library/jest-dom/extend-expect'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<Home />, div)
})

it('matches snapshot', () => {
  const tree = renderer.create(<Home />).toJSON()
  expect(tree).toMatchSnapshot()
})
