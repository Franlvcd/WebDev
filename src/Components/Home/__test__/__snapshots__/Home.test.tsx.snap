// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="home-background"
  >
    <div
      className="title-colour-layer"
    >
      <div
        style={
          Object {
            "display": "flex",
            "justifyContent": "center",
          }
        }
      >
        <div
          className="title-text"
        >
          <div
            className="title-background"
          >
            Care to make your solution more eco-friendly?
          </div>
          <div>
            Greener can help!
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
